python/cudf/cudf/core/series.py

```diff
@@ -399,17 +399,16 @@ def __init__(
         nan_as_null=True,
     ):
         if isinstance(data, pd.Series):
-            if name is None:
-                name = data.name
+            name = name or data.name
```

Comment:
We might have to explicitly check `name` against `None`, as `False` and `0`  are valid values for the `name` AFAIK.

Reply:
Good eye! Empty string `""` would also be falsy here.

Reply:
Fixed this.